controllers/books.js

```diff
@@ -4,42 +4,48 @@ let books = []
 
 
 export const getBooks = (req, res) => {
-  res.send(books);
-}
+      res.send(books);
+   }
 
-export const createBook = (req, res) =>{
-  const book = req.body;
+export const createBook = (req,    res)=>{
+   const book = req.body;
   
-   books.push({ ...book, id: uuidv4()});
-  
-  res.send(`Book with the name ${book.bookTitle} added to the database!`); }
+    books.push({ ...book, id: uuidv4()});
+    res.send(`Book with the name ${book.bookTitle} added to the database!`);
+   }
```

Comment:
Line 10 - remove unneeded spacing before "res" so it shows likes `(req, res) => {`
Lines 11,13,14,15 - indent properly